apps/discovery-platform/src/entry-server/routers/v1/index.spec.ts

```diff
@@ -256,10 +256,12 @@ describe("test v1 router", function () {
       await registeredNode.updateRegisteredNode(dbInstance, {
         ...firstCreatedNode.body.node!,
         status: "READY",
+        updated_at: new Date().toISOString(),
```

Comment:
Not sure how i feel about sending `updated_at` in every update instead of handling this inside the `updateRegisteredNode`. Maybe sending it on each function call can make it easy to test. 

what do you think @nionis ?